third-party/linshare/src/main/java/org/apache/james/linshare/client/UserResult.java

```diff
@@ -0,0 +1,89 @@
+package org.apache.james.linshare.client;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.List;
+import java.util.Objects;
+
+public class UserResult {
+
+    private final String uuid;
+    private final String creationDate;
+    private final Long modificationDate;
```

Comment:
is `null` a valid value?

Reply:
This is response from linshare so when user created it will not be null.

Reply:
If it cant be null, use `long` instead of `Long`